src/App.tsx

```diff
@@ -21,6 +22,7 @@ export default function App({ api }: { api: API }) {
   const [enabled, setEnabled] = useState(true);
   const [showConfetti, setShowConfetti] = useState(false);
   const [step, setStep] = useState<Step>("1:Welcome");
+  const { data: codeSnippets } = useGetProject();
```

Comment:
data is `null` initially. Is this case covered? Can the underlying components handle codeSnippets if it is null?

Reply:
Yup! They all have a safe check